### PR TITLE
Add a sentence and an example for custom log class

### PR DESCRIPTION
**Pull request by hogren**

### CODE REVIEW

File: doc/loggable.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ on how to setup and use the extensions in most optimized way.
 ### Loggable annotations:
 
 - **@Gedmo\Mapping\Annotation\Loggable(logEntryClass="my\class")** this class annotation
-will store logs to optionally specified **logEntryClass**. You will still need to specify versioned fields with the following annotation.
+will store logs to optionally specified **logEntryClass**. You will still need to specify versioned fields with the following annotation. If you decide to use a specific class, this class must be extend the ``Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry`` class.
 - **@Gedmo\Mapping\Annotation\Versioned** tracks annotated property for changes
```
Comment (phansys) [inline doc/loggable.md:37]:
```suggestion
will store logs to optionally specified **logEntryClass**. You will still need to specify versioned fields with the following annotation.
- **@Gedmo\Mapping\Annotation\Versioned** tracks annotated property for changes

**Note:** If you need to use a different class, it must extend ``Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry``.
```

File: doc/loggable.md [hunk 2]
```diff
@@ -197,6 +199,47 @@ class Article
...
+```php
+<?php
+
+namespace Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry;
+
+/**
+ *
```
Comment (phansys) [inline doc/loggable.md:216]:
```suggestion
```
BTW, try to mimic the structure defined by the `@DoctrineAnnotation` CS ruleset (see #2507).

**Review by phansys — COMMENTED** [review]
Congrats for your first contribution @hogren!

File: doc/loggable.md [hunk 3]
```diff
@@ -197,6 +199,46 @@ class Article
...
+
+/**
+ * @ORM\Table(
+ *     options={"row_format":"DYNAMIC"},
+ *     indexes={
+ *      @ORM\Index(name="log_class_lookup_idx", columns={"object_class"}),
+ *      @ORM\Index(name="log_date_lookup_idx", columns={"logged_at"}),
+ *      @ORM\Index(name="log_user_lookup_idx", columns={"username"}),
+ *      @ORM\Index(name="log_version_lookup_idx", columns={"object_id", "object_class", "version"})
+ *  }
```
Comment (phansys) [inline doc/loggable.md:223]:
```suggestion
 *         @ORM\Index(name="log_class_lookup_idx", columns={"object_class"}),
 *         @ORM\Index(name="log_date_lookup_idx", columns={"logged_at"}),
 *         @ORM\Index(name="log_user_lookup_idx", columns={"username"}),
 *         @ORM\Index(name="log_version_lookup_idx", columns={"object_id", "object_class", "version"})
 *     }
```

File: doc/loggable.md [hunk 4]
```diff
@@ -197,6 +199,46 @@ class Article
 </doctrine-mapping>
 ```
 
+
```
Comment (phansys) [inline doc/loggable.md:202]:
```suggestion
```

File: doc/loggable.md [hunk 5]
```diff
@@ -197,6 +199,46 @@ class Article
...
+#[ORM\Index(name: 'log_version_lookup_idx', columns: ['object_id', 'object_class', 'version'])]
+class ParameterHistory extends AbstractLogEntry
+{
+    /*
+     * All required columns are mapped through inherited superclass
+     */
+}
+```
+
+
```
Comment (phansys) [inline doc/loggable.md:241]:
```suggestion
```

**Review by phansys — APPROVED** [review]

**Comment by franmomu** [comment]
Thanks @hogren!

**Pull request merged**